src/pip/_internal/cache.py

```diff
@@ -231,8 +231,10 @@ class WheelCache(Cache):
     when a certain link is not found in the simple wheel cache first.
     """
 
-    def __init__(self, cache_dir, format_control):
-        # type: (str, FormatControl) -> None
+    def __init__(self, cache_dir, format_control=None):
+        # type: (str, Optional[FormatControl]) -> None
+        if format_control is None:
+            format_control = FormatControl()
```

Comment:
Why not explicitly pass a fresh `FormatControl()` from the caller instead?

Reply:
Ultimately the wheel cache will not be concerned about format control at all, so I wanted to already expose a constructor without FormatControl.

Reply:
Our cache logic is very tied to the FormatControl class, and I'd prefer to make this _not_ make this optional. Having this be an explicit dependency was an intentional choice, to make it clearer what the object needs to function.

Reply:
Further thinking about this I'll see if I can make this PR not touch the caching use logic. Contrarily to what I initially thought it might be possible, and we can revisit the cache logic independently (which is the topic of https://github.com/pypa/pip/issues/9162).